### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by vxbinaca**
- `apt` should be used over `apt-get`
- Set execute bit on script.
- Updated README to instruct to call script outside of python
- Converted some commands to one-liners
- Instructed to update system software

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The upload is enriched with metadata from the github api and the README.md
 
 Prerequisites (with debian or ubuntu):
 
+    apt update && apt upgrade
```
Comment (gdamdam) [inline README.md]: the update of all the installed packages is not a requirement for installing iagitup.

**Review by gdamdam — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vxbinaca) [reply README.md]: Alright drop this commit then and keep the rest.

**Comment by vxbinaca** [comment]
I'm told you were inspired by Tubeup? I didn't write it but I help maintain it. All I did was improve the library calls for youtube-dl for stability and guaranteed best quality on the files (also subtitle ripping), improve the opts for internetarchive for stability and just evangelize it.

This script is neat though and I want to use it to start backing up software that's at risk of being removed.

**Comment by gdamdam** [comment]
thank you @vxbinaca.
iagitup is indeed strongly inspired by Tubeup, that is an amazing tool for archiving video!
I want to write a similar tool for the code and git repositories.

Please feel free to use the script and let me know how to improve it!

**Comment by vxbinaca** [comment]
I will, and that goes for you too. If theres an improvement you want seen to tubeup submit a PR.

**Pull request merged**